src/main/java/com/petfinder/controller/ExampleController.java

```diff
@@ -1,18 +1,40 @@
 package com.petfinder.controller;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 
+import com.petfinder.exception.EmailExistsException;
+import com.petfinder.exception.InvalidEmailException;
+import com.petfinder.exception.LoginExistsException;
+import com.petfinder.service.UserService;
+
 @Controller
 @RequestMapping("/example")
 public class ExampleController {
 	
+	@Autowired
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matikz1110/PetFinder/file/2508688999/issues/source?bid=3239464&fileBranchId=3276039#l19)